test/connection.js

```diff
@@ -39,13 +40,22 @@ describe('testConnection', function() {
 });
 
 function generateURL(config) {
-  var url =
-    'mssql://' + config.user +
-    ':' + config.password +
-    '@' + config.host +
-    ':' + config.port +
-    '/' + config.database +
-    '?encrypt=true';
-  console.log('My url is>>>', url);
+  var urlObj = {
+    hostname:config.host,
+    auth:config.user + ':' + config.password,
+    protocol: 'mssql',
+    pathnname: config.database,
```

Comment:
Ah, you have already fixed this, ignore my comment here - I'll delete it right now.